modules/thermal_hydraulics/src/materials/ADWallHeatTransferCoefficient3EqnDittusBoelterMaterial.C

```diff
@@ -34,7 +37,7 @@ ADWallHeatTransferCoefficient3EqnDittusBoelterMaterial::validParams()
 ADWallHeatTransferCoefficient3EqnDittusBoelterMaterial::
     ADWallHeatTransferCoefficient3EqnDittusBoelterMaterial(const InputParameters & parameters)
   : Material(parameters),
-    _Hw(declareADProperty<Real>(FlowModelSinglePhase::HEAT_TRANSFER_COEFFICIENT_WALL)),
+    _Hw(declareADProperty<Real>("Hw")),
```

Comment:
TIL we don't have to do `declareProperty<Real>(getParam<MaterialPropertyName>("Hw"))`. I thought it was only like that for getting properties, but I checked, and it's not!

Reply:
Also, did you test the non-AD one locally? I think you need the same change there. We'll find out after testing finishes!

Reply:
Yeah the test is fine locally. But you are right, I'd think the non-AD one would require it as well. Maybe the test I added is not testing that properly.

Reply:
Oh, you know what? It's because `FlowModelSinglePhase::HEAT_TRANSFER_COEFFICIENT_WALL` happens to translate to `"Hw"`... Anyway, that'll need to get fixed.